include/game.h

```diff
@@ -1726,7 +1729,7 @@ typedef struct {
 
     /* 80072F24 */ s32 unk04; // copy of the previous field
     /* 80072F28 */ s32 unk08;
-    /* 80072F2C */ u32 unk0C;
+    /* 80072F2C */ PlayerStateStatus status;
```

Comment:
I don't have particularly strong feelings either way, but IMO a small advantage here is using an enumeration might be more self-documenting code than the #define approach.

Doesn't look like we get anything in the way of compile time help by doing this so I'm fine reverting this to #define if preferred.